CMakeLists.txt

```diff
@@ -103,6 +108,21 @@ set(bullet_GIT_REPOSITORY https://github.com/RobotLocomotion/bullet-pod.git)
 set(bullet_GIT_TAG 54d0751b50f86481549391961b209f06ac5035f9)
 set(bullet_IS_PUBLIC TRUE)
 set(bullet_IS_CMAKE_POD TRUE)
+set(libccd_GIT_REPOSITORY https://github.com/danfis/libccd.git)
+set(libccd_GIT_TAG v2.0)
+set(libccd_IS_PUBLIC TRUE)
+set(libccd_IS_CMAKE_POD TRUE)
+set(flann_GIT_REPOSITORY https://github.com/mariusmuja/flann.git)
+set(flann_GIT_TAG 1.8.1-src)
+set(flann_IS_PUBLIC TRUE)
+set(flann_IS_CMAKE_POD TRUE)
+#set(fcl_GIT_REPOSITORY https://github.com/flexible-collision-library/fcl.git)
+#set(fcl_GIT_TAG 0.4.0)
+set(fcl_GIT_REPOSITORY https://github.com/law12019/fcl.git)
```

Comment:
Why did you have to make your own fork? Is there a change that could be pushed back to the upstream fork so it could be used instead?